benchmarks/templates/benchmarks/landing_page.html

```diff
@@ -1,11 +1,165 @@
+{% endcompress %}
+
+{% block navbar %}
+<nav class="navbar updated is-fixed-top is-transparent" role="navigation" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <div class= "container is-full-fullhd">
+        <div class="navbar-brand">
+            <a class="navbar-item" href="https://www.brain-score.org">
+                    <img class="navbar_element image" id="navbar-logo-colored" src="/static/benchmarks/img/new_ui_elements/colored_logo.png"/>
+            </a>
+            <a role="button" class="navbar-burger" data-target="navMenu" aria-label="menu" aria-expanded="false">
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+            </a>
+        </div>
+        <div id="navMenu" class="navbar-menu">
+                <div class="navbar-end">
+                    <div class="navbar-item">
+                        <button onclick="location.href='../profile'" type="button" class="button btn-1" >Log In</button>
+                        <a class="social_icon navbar-item " href="https://www.github.com/brain-score">
+                            <img class="navbar_element image" id="github_icon" src="/static/benchmarks/img/new_ui_elements/github_icon.png" />
+                        </a>
+                        <a class="social_icon navbar-item" href="https://www.twitter.com/brain_score">
+                            <img class="navbar_element image" id="twitter_icon" src="/static/benchmarks/img/new_ui_elements/twitter_icon.png"/>
+                        </a>
+                    </div>
+                </div>
+        </div>
+    </div>
+</nav>
+{% endblock %}
+
+<div class="landing container">
+    <div class="columns">
+        <div class="left_side column is-half has-text-centered-mobile">
+            <img class="logo_green image is-inline-block"  src="/static/benchmarks/img/new_ui_elements/logo_name_full.png" />
+            <h3 class="landing_heading is-size-6 is-size-4-widescreen is-size-4-mobile">Integrative Benchmarks | Models at Scale </h3>
+            <p class="text-default-small is-size-7 is-size-6-widescreen has-text-justified-mobile">The Brain-Score platform aims to yield strong computational models of the
+                brain and mind. We enable researchers to quickly get a sense of the alignment of their model(s) to
+                currently dozens of neural and behavioral measurements, and provide these
+                models to experimentalists to prototype new experiments and make sense of biological data. </p>
+            <div class="domain_buttons container">
+                <button onclick="location.href='../vision'" type="button" class="button domain_button is-size-7 is-size-6-widescreen" >Vision</button>
+                <button onclick="location.href='../language'" type="button" class="button domain_button is-size-7 is-size-6-widescreen" >Language</button>
```

Comment:
The `../` in front of `vision` and `language` concerns me, why are those necessary?